src/sdk/python/pkg/setup.py

```diff
@@ -15,7 +15,7 @@
 
 setup(
     name="boschio-iotea",
-    version="0.3.0",
+    version="0.4.0",
```

Comment:
Are the changes of this version somewhere documented?

Reply:
I will add the points to the CHANGELOG.md document, which is already accompanied within the repo